build-unified.nu

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nu
+# generates modules-latest directory with only latest versions of modules and builds the Containerfile
+
+print $"(ansi green_bold)Gathering images"
+
+rm -rf ./modules-latest
+mkdir ./modules-latest
+
+ls modules | each { |moduleDir|
+
+    # module is unversioned
+    if ($"($moduleDir.name)/($moduleDir.name | path basename).sh" | path exists) {
+
+        print $"(ansi cyan)Found(ansi reset) (ansi cyan_bold)unversioned(ansi reset) (ansi cyan)module:(ansi reset) ($moduleDir.name | path basename)"
+
+        cp --recursive ($moduleDir.name) $"./modules-latest/($moduleDir.name | path basename)"
+
+    } else { # module is versioned
+
+        print -n $"(ansi cyan)Found(ansi reset) (ansi blue_bold)versioned(ansi reset) (ansi cyan)module:(ansi reset) ($moduleDir.name | path basename), "
+
+        let latest = glob $"./($moduleDir.name)/v*" | last # the glob result is already orderer such that the last value is the biggest
+
+        print $"(ansi blue_bold)Latest version:(ansi reset) ($latest | path basename)"
+
+        cp --recursive ($latest) $"./modules-latest/($moduleDir.name | path basename)"
+
+    }
+}
+
+print $"(ansi green_bold)Starting image build"
```

Comment:
And here. Does it just reset at the end of a print anyways? If so you can ignore all the comments

Reply:
I'm not sure, but I didn't encounter any issues with it. I can still add the resets, though.